calculator/lexer.py

```diff
@@ -1,4 +1,65 @@
-def lex(s):
+ops = ["+", "-", "*", "/"]
+ops_without_subtraction = ["+", "*", "/"]
+
+
+def is_numeric_char(char):
+    """
+    is_numeric_char checks if a char is a number or a decimal point.
+    Input: char 
+    Output: True or False
+    """
+    if char.isdigit() or char == ".":
```

Comment:
simply `return char.isdigit() or char == "."` please.  No need for the if/else

Reply:
👍